### ISSUE TITLE
3.0.2 missing settings

### ISSUE THREAD

**Issue by Unknown-Zombie** [Marginal/EDMarketConnector#311]
I updated to 3.0.2 and the Configuration, Appearance, and Plugins tabs are all missing from the Settings window.

Also in 3.0.2 File > Status does nothing; clicking it will not open the window it's supposed to with your recorded stats.

I uninstalled and reinstalled 3.0.2 but this didn't solve the problem, so I uninstalled again and went back to using 3.0.1

**Comment by robert914** [comment]
I'm seeing the same problem. The EDMarketConnector.log in temp has this:

```
E:D Market Connector 3.0.2.0 2018-04-04T16:42:45
loading plugin eddb from "D:\Program Files\EDMarketConnector\plugins\eddb.py"
loading plugin edsm from "D:\Program Files\EDMarketConnector\plugins\edsm.py"
loading plugin inara from "D:\Program Files\EDMarketConnector\plugins\inara.py"
Exception in Tkinter callback
Traceback (most recent call last):
  File "Tkinter.pyo", line 1542, in __call__
  File "EDMarketConnector.py", line 190, in <lambda>
  File "prefs.pyo", line 247, in __init__
  File "myNotebook.pyo", line 148, in __init__
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```

**Comment by Marginal** [comment]
Sorry about that. Please do Help -> Check for Updates again.

**Comment by robert914** [comment]
Seems ok now! Thanks for the fast turn-around!

**Comment by Unknown-Zombie** [comment]
All the settings tabs are back, but File > Status still isn't working.

Here's the logfile:

```
E:D Market Connector 3.0.2.1 2018-04-04T18:52:31
loading plugin coriolis from "C:\Program Files (x86)\EDMarketConnector\plugins\coriolis.py"
loading plugin eddb from "C:\Program Files (x86)\EDMarketConnector\plugins\eddb.py"
loading plugin edsm from "C:\Program Files (x86)\EDMarketConnector\plugins\edsm.py"
loading plugin edsy from "C:\Program Files (x86)\EDMarketConnector\plugins\edsy.py"
loading plugin inara from "C:\Program Files (x86)\EDMarketConnector\plugins\inara.py"
loading plugin CGThresholds from "C:\Users\james\AppData\Local\EDMarketConnector\plugins\CGThresholds\load.py"
CGPlugin Starting
Exception in Tkinter callback
Traceback (most recent call last):
  File "Tkinter.pyo", line 1542, in __call__
  File "EDMarketConnector.py", line 188, in <lambda>
  File "stats.pyo", line 178, in __init__
AttributeError: AppWindow instance has no attribute 'session'
Exception in Tkinter callback
Traceback (most recent call last):
  File "Tkinter.pyo", line 1542, in __call__
  File "EDMarketConnector.py", line 188, in <lambda>
  File "stats.pyo", line 178, in __init__
AttributeError: AppWindow instance has no attribute 'session'
Exception in Tkinter callback
Traceback (most recent call last):
  File "Tkinter.pyo", line 1542, in __call__
  File "EDMarketConnector.py", line 188, in <lambda>
  File "stats.pyo", line 178, in __init__
AttributeError: AppWindow instance has no attribute 'session'
```

**Comment by robert914** [comment]
FYI, I can't get the Mac client to start either at all. It doesn't even generate a log.

**Comment by Marginal** [comment]
@Unknown-Zombie Yep, I broke it. Will be fixed in the next release.
@robert914 I don't see any problem. What happens? What version of macOS? The log file is at `$TMPDIR/EDMarketConnector.log`.

**Comment by robert914** [comment]
I try to open it and I immediately get a dialog stating "E:D Market Connector Error" with buttons "Open Console" or "Terminate". There is no log created in $TMPDIR as I mentioned before (I knew that's where it went normally). I'm using MacOSX 10.12.6.  I've attached a screenshot of the dialog:
![errordialog](https://user-images.githubusercontent.com/214286/38373790-e44abf52-38a6-11e8-8e5e-a536994f3481.png)

**Comment by Marginal** [comment]
@robert914 I can't reproduce the problem on 10.12.6.
Assuming you've put the app in /Applications, please open the Terminal app, copy and paste the following into it:
```
"/Applications/EDMarketConnector.app/Contents/MacOS/E:D Market Connector"
```
and press Enter.

**Comment by robert914** [comment]
Ok, found the problem running it that way, I had a symbolic link for "plugins" to another location, and that caused it to spit some python errors about "File already exists" when it apparently tried to create the plugins folder since it thought it was not an actual folder. I removed my symbolic link and put back the directory instead, and now it works.

**Comment by Marginal** [comment]
File -> Status fixed in release 3.03.
